Add stopped icon option

DimmKirr opened the pull request:
# Changes
Adds an option to customize output when there is no player running. This is a backwards-compatible change.

# Reasoning
I like to see `⏹︎` instead of an empty space in my permanent status bar section when no player is running. Reminds me to turn on some music.

<img width="269" height="34" alt="image" src="https://github.com/user-attachments/assets/58e13958-bdbd-4cad-ad47-08dd60691b4a" />

spywhere commented on README.md:
```diff
@@ -46,6 +46,10 @@ Description: A string to display for `{icon}` component when the music player
 is paused  
 Default: ` ` (empty space)  
 Values: string
+- `@now-playing-stopped-icon`  
```
suggestion: instead of calling it 'stopped', may be better name it as `@now-playing-idle-icon`?

Idle and stopped can have a different meaning

- Idle: No music player running
- Stopped: No music playing (but the player might still running)

spywhere commented on scripts/music.sh:
```diff
@@ -113,6 +115,17 @@ main() {
 
+
+  # Check if track duration is 0 (state in-between a "play/pause" and "no-player")
+  if test "$track_duration" -eq 0; then
```
suggestion(optional): for this one, I would preferred to have another state, 'stopped'.

See [AppleScript player](https://github.com/spywhere/tmux-now-playing/blob/master/players/applescript.js#L57) implementation here, it does produce a stopped state. However, this might not be for all players.

If you are not comfortable to update it, feel free to skip this one.

spywhere reviewed (changes requested):
Thanks for the PR and sorry for a late review. Just a few comments, then it should be all good.

spywhere commented on README.md:
```diff
@@ -46,6 +46,10 @@ Description: A string to display for `{icon}` component when the music player
 Values: string
+- `@now-playing-stopped-icon`  
+Description: A string to display for `{icon}` component when there is no music player running  
+Default: ` ` (empty space)  
```
suggestion: let's also default it to empty string? To keep the behavior as-is.

DimmKirr commented:
Hi @spywhere thanks for taking time to review it. Will address the notes and ping you back

dmitry-kryuk committed:
Add stopped and idle icons for music player states
- feat(readme): document now-playing-stopped-icon and now-playing-idle-icon options
- feat(scripts/music): show idle icon when no player running and stopped icon when player stopped
- feat(scripts/music): update icon selection logic to include stopped and idle states

DimmKirr commented:
@spywhere OK made changes please review when you have a moment

spywhere commented:
Cool, thanks for the contribution.
